JsonRocket/JsonRocket.csproj

```diff
@@ -11,4 +11,8 @@
     <PackageTags>json</PackageTags>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard1.3|AnyCPU'">
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
actually, do we need unsafe at all ? i haven't seen any unsafe code so far ...

Reply:
yeah we don't, will remove the unsafe stuff